client/state/i18n/locale-suggestions/reducer.js

```diff
@@ -0,0 +1,21 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { I18N_LOCALE_SUGGESTIONS_ADD } from 'state/action-types';
+
+import { combineReducers } from 'state/utils';
+
+export const items = ( state = null, action ) => {
+	switch ( action.type ) {
+		case I18N_LOCALE_SUGGESTIONS_ADD:
+			return action.items;
+		default:
+			return state;
+	}
+};
+
+export default combineReducers( {
+	items,
+} );
```

Comment:
If we don't think this is going to grow we could also just export `items` as the default otherwise we're introducing an additional level of indirection in the state tree with no gain.